action.yml

```diff
@@ -6,8 +6,7 @@ runs:
   main: "./lib/index.js"
 inputs:
   formula-name:
-    description: The name of the Homebrew formula
-    required: true
+    description: The name of the Homebrew formula (defaults to the repo portion of ${{ github.repository }})
```

Comment:
Is `${{ github.repository }}` even expanded in a `description:` field? Let's just say "defaults to lower-cased repository name"